pallets/subtensor/src/macros/dispatches.rs

```diff
@@ -0,0 +1,846 @@
+use frame_support::pallet_macros::pallet_section;
+
+/// A [`pallet_section`] that defines the errors for a pallet.
+/// This can later be imported into the pallet using [`import_section`].
+#[pallet_section]
+mod dispatches {
```

Comment:
can we reset all the extrinsic's call_index, make them in order. don't know how to choose the next call index when add a new extrinsic